### PR TITLE
Global styles: move preload paths filter to 6.7 (previously 6.8)

### PR DESCRIPTION
**Pull request by ramonjd**
## What?

Follow up to:

- https://github.com/WordPress/gutenberg/pull/66468

https://github.com/WordPress/gutenberg/pull/66468 will be [cherry picked to 6.7](https://github.com/WordPress/gutenberg/pull/66542) since it's a regression.

## Why?
Given that this fixes a 6.7 regression move the code into 6.7 compat.

## How?
Copy paste

## Testing Instructions

Fire up this branch and head to the site editor.

Check that global-styles endpoints aren't fetched client side, but are, rather, preloaded via [rest_preload_api_request](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api.php#L2918)

Do the same for the post editor.

### CODE REVIEW

File: backport-changelog/6.8/7661.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 https://github.com/WordPress/wordpress-develop/pull/7661
 
 * https://github.com/WordPress/gutenberg/pull/66468
+* https://github.com/WordPress/gutenberg/pull/66543
```
Comment (aaronrobertshaw) [inline backport-changelog/6.8/7661.md:4]: I think this might need moving to the 6.7 backport-changelog folder too.

**Review by andrewserong — APPROVED** [review]
Thanks for the follow-up, this is testing well for me!

Looks like the only change is `$stylesheet` being renamed to `$active_theme` 👍 

Once the backport `.md` file is moved to 6.7, this looks good to land to me!

**Review by aaronrobertshaw — APPROVED** [review]
LGTM!

Global styles paths are still preloaded correctly.

🚢

**Comment by ramonjd** [comment]
Thanks for the quick reviews, folks!

**Pull request merged**